src/api/transfer.js

```diff
@@ -13,7 +13,7 @@ const validateInputs = validateWithJoi(schema)('INVALID_METHOD_ARGUMENT')({
   context: `transfer`
 })
 
-module.exports = async (dvf, data, starkPrivateKey, nonce, signature) => {
+module.exports = async (dvf, data, nonce, signature) => {
```

Comment:
isn't this change going to break any usage of this function/

Reply:
Yes this feature now requires a provider to be initialized before calling it.
`starkPrivateKey` was not used anymore so we are removing it from the arguments list.
The portal will need to be updated to follow this new signature once this PR is merged.

Reply:
Do we need to also bump a major version change on this repo as API user's using this function will also need to make changes?

Reply:
True. Would `2.0.0` make sense as new version number since those are effectively breaking changes ?

Reply:
indeed, I'm unsure of the versioning practices we follow here, @konradstrachan or @Matko95 would be best to answer that question. We can also create a new method and put a deprecate flag on the old one